index.js

```diff
@@ -34,7 +34,9 @@ var getFiles = function(options) {
 };
 
 var getOptions = function() {
-  var globalOptions = this.options.elm || {};
+  var globalOptions = this.options
+    ? this.options.elm || {}
+    : this.query.elm || {};
```

Comment:
Do we really want this? Is this.query supported in older versions of webpack?

Reply:
logging the properties whilst building from examples:
v1 supports this.options
v2 supports both
v4 supports this.query

Reply:
:+1: